src/wiktextract/extractor/simple/page.py

```diff
@@ -0,0 +1,125 @@
+        logger.info(f"Parsing page: {page_title}")
+
+    wxr.config.word = page_title
+    wxr.wtp.start_page(page_title)
+
+    #####  Debug printing #####
+    # Temporary debug print stuff without page parsing goes into debug_bypass.
+    # If you're running a full extraction to find data, multiprocessing
+    # *will* sometimes mess up your prints by overlaying some prints
+    # with others. For quick and dirt stuff this might not matter, but if
+    # you really want to be sure to get good prints you need to use
+    # something like the logging package, which here is represented by
+    # `logger`; the only annoyance is that it's not easy to get rid of
+    # the datetime string at the start on the fly, so I just do it crudely
+    # by inserting a `\n` in the message after `f"{wxr.wtp.title}..."`.
+
+    # from .debug_bypass import debug_bypass
+    # return debug_bypass(wxr, page_title, page_text)
+
+    #####
+
+    ##### Main page parse #####
+    page_root = wxr.wtp.parse(
+        page_text,
+    )
+
+    # print_tree(page_root)
+
+    # If this was a normal wiktionary edition, this is where you would split
+    # the page into different language entries; "English", "Gaelic", "Swahili",
+    # etc.
+    # Each section would be a Level 2 node (`NodeKind.LEVEL2`) child of the
+    # page's root node. Wiktionaries do not seem to use LEVEL1 for anything,
+    # although there might be (there definitely is...) an exception somewhere.
+
+    # Some data that is collected is shared among several entries; things like
+    # pronunciation data and etymological data is collected and updated in this
+    # base_data object that can be copied for more specific entries.
+    # Simple English Wiktionary has these sections under Level 3 ("===")
+    # headers, which put them in an awkward hierarchy with the main page and
+    # Part-of-Speech sections, which are higher; they usually appear before
+    # POS sections, but a few pages have a more complex structure. This
+    # is handled by simply flattening the parse tree later, and handling
+    # sections in a linear order.
+    base_data = WordEntry(
+        word=wxr.wtp.title or "ERROR_NO_TITLE",
+        # Simple English wiktionary entries are always for English words
+        lang_code="en",
+        lang="English",
+        pos="ERROR_UNKNOWN_POS",
+    )
+
+    word_datas: list[WordEntry] = []
+
+    for level in page_root.find_child_recursively(LEVEL_KIND_FLAGS):
+        # Ignore everything outside of a section with a heading; there shouldn't
+        # be anything there. Previous version of this code looked through that
+        # stuff and would spit out warnings.
+
+        # .find_child_recursively() (in contrast with find_child()) yields
+        # a flattened tree, not just direct children of root
+
+        # print(f"=== {heading_title=}")
+        heading_title = clean_node(wxr, None, level.largs[0]).lower()
+
+        if m := re.search(r"\s+\d+h$", heading_title):
```

Comment:
typo?
```suggestion
        if m := re.search(r"\s+\d+$", heading_title):
```

Reply:
Oh crap. Yeah, that's a typo... I'm using a modal editor at the moment with heavily rebound keys (HJKL is a terrible historical accident, ask me about it), "h" is my "insert text" key.

Reply:
I used to use vim, but I'm too lazy to customize any keys even now... I vaguely remember that's cursor movement keys, and "i" for insert?